README.md

```diff
@@ -1,2 +1,162 @@
 # trampss-redux-data-store
+
+Factory of Redux reducers and their associated actions and selectors.
+> Make your Redux code base tinier and simpler to maintain
+
 [![CircleCI](https://circleci.com/gh/Trampss/trampss-redux-data-store.svg?style=shield)](https://circleci.com/gh/Trampss/trampss-redux-data-store) [![Coverage Status](https://coveralls.io/repos/github/Trampss/trampss-redux-data-store/badge.svg?branch=master)](https://coveralls.io/github/Trampss/trampss-redux-data-store?branch=master) [![NPM Version](https://badge.fury.io/js/trampss-redux-data-store.svg)](https://www.npmjs.com/package/trampss-redux-data-store)
+
+## Purpose
+`trampss-redux-data-store` generates generics reducers, actions and selectors in two line.
+
+```es6
+import factory from 'trampss-redux-data-store'
+export default factory('id')('api')('todos')
+```
+That's it, you exported a function that is a reducer to include in a Redux `combineReducer`.
+The exported function also have actions and selectors ! Easy.
+
+In this example, we have a `todos` reducers, it has to be combined into `state.api.todos`
+
+## Why
+By using `redux-saga` our Redux code base was simpler every day : it is almost a key/value store right now. But we had plenty of duplicate code (each of our ressource has its reducers, its actions, its selectors).
+
+To avoid growing Redux code base, and to avoid inconsistency, and pain maintenability, we created this little library that is a factory of reducers, actions and selectors.
```

Comment:
what about `tiny` ? I would like to advertise the fact that is <4kb (https://github.com/Trampss/trampss-redux-data-store/blob/master/index.js)